legal-api/tests/unit/services/warnings/business/business_checks/test_involuntary_dissolution.py

```diff
@@ -0,0 +1,93 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in business with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test suite to ensure Corpse business checks work correctly."""
+import copy
+import pytest
+import json
+from legal_api.models import Batch, Business
+from tests.unit.models import (
+    factory_batch,
+    factory_batch_processing,
+    factory_business,
+    factory_completed_filing,
+    factory_pending_filing,
+)
+from legal_api.services.warnings.business.business_checks import WarningType
+from legal_api.services.warnings.business.business_checks.involuntary_dissolution import check_business
+from legal_api.utils.datetime import datetime
+
+from datedelta import datedelta
+from registry_schemas.example_data import FILING_HEADER, RESTORATION, TRANSITION_FILING_TEMPLATE
+
+RESTORATION_FILING = copy.deepcopy(FILING_HEADER)
+RESTORATION_FILING['filing']['restoration'] = RESTORATION
+
+@pytest.mark.parametrize('test_name, no_dissolution, batch_status, batch_processing_status', [
+    ('NOT_ELIGIBLE', True, None, None),
+    ('ELIGIBLE', False, 'PROCESSING', 'COMPLETED'),
```

Comment:
can you rename this test to `ELIGIBLE_AR_OVERDUE`?

Reply:
Updated